server.go

```diff
@@ -1755,11 +1756,7 @@ func (s *server) peerTerminationWatcher(p *peer) {
 	}
 
 	for _, link := range links {
-		err := p.server.htlcSwitch.RemoveLink(link.ChanID())
-		if err != nil {
-			srvrLog.Errorf("unable to remove channel link: %v",
-				err)
-		}
+		p.server.htlcSwitch.RemoveLink(link.ChanID())
```

Comment:
Commit message looks to not actually match what's in the commit itself.

Reply:
Fixed

Reply:
Not blocking, but still looks diff, it says: 
> server: use blocking RemoveLink to shutdown links

Yet it just now ignores the error.

Reply:
`RemoveLink` doesn't return an error. It just blocks until everything is purged